### PR TITLE
Revert "Update controller-start-up.md:- Corrects a few grammatical errors"

### PR DESCRIPTION
**Pull request by PRUBHTEJ**
Reverts cyberbotics/webots#1692

### CODE REVIEW

File: docs/guide/controller-start-up.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 The .wbt file contains the name of the controller that needs to be started for each robot.
 The controller name is a platform and language independent field; for example when a controller name is specified as "xyz\_controller" in the .wbt file, this does not say anything about the controller's programming language or platform.
-This is done deliberately to ensure the independence between platform and programming language of .wbt files.
+This is done deliberately to ensure the platform and programming language independence of .wbt files.
```
Comment (omichel) [inline docs/guide/controller-start-up.md:5]:
```suggestion
This is done deliberately to ensure that the *.wbt* file is independent from the platform and programming language.
```

File: docs/guide/controller-start-up.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 
-Now, when Webots tries to start a controller it must determine what programming language is used by that controller initially.
-So, Webots looks in the project's "controllers" directory for a subdirectory that matches the controller's name.
+So when Webots tries to start a controller it must first determine what programming language is used by this controller.
```
Comment (omichel) [inline docs/guide/controller-start-up.md:7]:
```suggestion
When Webots tries to start a controller it must first determine what programming language is used by this controller.
```

File: docs/guide/controller-start-up.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
+So, Webots looks in the project's "controllers" directory for a subdirectory that matches the controller name.
```
Comment (omichel) [inline docs/guide/controller-start-up.md:8]:
```suggestion
So, Webots looks in the project's *controllers* directory for a subdirectory that matches the name of the controller.
```

File: docs/guide/controller-start-up.md (hunk 2 above)
Comment (PRUBHTEJ) [reply docs/guide/controller-start-up.md:7]: Thanks for your guidance and help, sir !!

**Pull request merged**